Lint Thai for real ellipsis

mattleff opened the pull request:
Fixes #269.

bountonw commented on lo/GC/001_machineraw/GC21_lo.md:
@mattleff are we sure we have the code right? It changed `....` to `….` I didn't check further. But I know there are some ."... and !... etc. in the mix.

bountonw reviewed (commented):
@mattleff probably better to comment here instead of in the file. 

I'm currently working on translating GC29. In this file I see `:....` `;....`

On second thought. Ignore the Lao files. They will need to be handled differently since Lao uses the full range of English punctuation. For Thai we need to strip out everything but the ellipsis and I think we need to standardize to the three dot ellipsis character (since Thai doesn't have sentence ending punctuation.)

mattleff commented:
@bountonw I've changed this PR to only lint Thai for real ellipsis.

mattleff commented:
@bountonw Bump for review. This PR only changes Thai files now. I need this on main or I'll have to revert the lint commit in the tooling repo for now.

bountonw commented:
There are inline comments from my previous review. I don't see the changes.

mattleff commented:
@bountonw I removed all the changes to the Lao files which is where we had the ellipsis stacked with other punctuation (such as `….` or `…!`). I also changed the linting to only apply to Thai files (for now).

bountonw commented:
@mattleff Did you see my inline comments?

mattleff commented:
@bountonw I think so, regarding changing `....` to `….`, right? But Thai doesn't have any punctuation use with ellipsis and I'm only dealing with Thai on this PR.

bountonw commented on th/PP/PP5/02_edit/PP62_th.md:
```diff
@@ -14,7 +14,7 @@ chapter:
 
 ## {PP 637.2}
 
-ในขณะที่ดาวิดใช้ชีวิตสมถะเรียบง่ายตามประสาผู้เลี้ยงแกะอยู่นั้น พระเจ้าตรัสกับผู้เผยพระวจนะซามูเอลเกี่ยวกับเขา “องค์พระผู้เป็นเจ้าตรัสกับซามูเอลว่า ‘เจ้าจะคร่ำครวญเรื่องที่เราถอดซาอูลออกจากตำแหน่งกษัตริย์อิสราเอลไปอีกนานเท่าใด บัดนี้จงเติมน้ำมันให้เต็มเขาสัตว์ และออกเดินทาง เราจะส่งเจ้าไปพบเจสซีแห่งเบธเลเฮม เราได้เลือกบุตรชายคนหนึ่งของเขาให้เป็นกษัตริย์’... ‘จงเอาวัวตัวเมียไปด้วยตัวหนึ่ง และพูดว่า ...
```
Here is another example of a space after the ellipsis. This is different from the other issues.

bountonw reviewed (commented):
@mattleff there are unresolved I line comments. Basically there are spaces before, after, or surrounding some ellipses. This needs to be standardized.

mattleff commented:
@bountonw Ah, I see a bunch of comments now. Ok, I can work on this. Are we wanting to eliminate all spaces before and after ellipsis, or is it more complicated?

bountonw commented:
I think we want to eliminate all of them. Is there an exception?

mattleff commented:
@bountonw Check out ba69a32 and make sure this is looking right to you. I'll resolve merge conflicts if the PR looks good to go otherwise.

bountonw commented:
@mattleff Looks good.
